packages/compat/tests/addon-styles.test.ts

```diff
@@ -95,11 +95,11 @@ describe('addon.styles tests', function () {
 
   test(`all addon CSS gets convert to implicit-styles`, function () {
     let implicitStyles = expectFile('node_modules/my-addon3/package.json').json().get('ember-addon.implicit-styles');
-    implicitStyles.includes('./my-addon3.css');
-    implicitStyles.includes('./outer.css');
-    implicitStyles.includes('./nested/inner.css');
-    expectFile('node_modules/my-addon3/my-addon3.css').matches(`from-addon`);
-    expectFile('node_modules/my-addon3/outer.css').matches(`from-outer`);
-    expectFile('node_modules/my-addon3/nested/inner.css').matches(`from-inner`);
+    implicitStyles.includes('./my-addon3/__COMPILED_STYLES__/my-addon3.css');
```

Comment:
I need to confirm this is ok.

Reply:
Given that the files exists here, and are that the tests [import files at the new location](https://github.com/embroider-build/embroider/pull/925/files#diff-d60754db9ebc16a05d82a04bff616a0036594a63fa5e2efafb1db1429ef2ea7aR721) it should be connected correctly..

Reply:
No, I think this is a bug caused by the ember-cli upgrade. We don't want to see namespacing here, these are supposed to be paths inside the addon, and shouldn't include `./my-addon3/` in the path.

Reply:
@ef4 TL;DR ember-cli bug or something you believe embroider needs to be aware of

Reply:
I'll attempt to decouple that version of ember-cli for the effected package from this PR, and investigate more deeply independently.